YubiKitDemo/YubiKitDemo/UI/Other/Demos/FIDO2DemoViewController.swift

```diff
@@ -511,6 +508,45 @@ class FIDO2DemoViewController: OtherDemoRootViewController {
         if sessionState == .closed {
             logTextView.text = nil
             setDemoButtons(enabled: true)
+        } else if sessionState == .open {
+            if YubiKitDeviceCapabilities.supportsISO7816NFCTags {
+                guard #available(iOS 13.0, *) else {
+                    fatalError()
+                }
+            
+                DispatchQueue.global(qos: .default).async { [weak self] in
```

Comment:
JFYI, I've seen a recommendation to not use .default global queue: ".default: don’t use it usually, the type will be inferred by the system".
Here is the article:  https://hackernoon.com/swift-multi-threading-using-gcd-for-beginners-2581b7aa21cb

Reply:
Reading official apple documentation I haven't found anything agains using it.
https://developer.apple.com/documentation/dispatch/dispatchqos/qosclass/default
And this task is actually seems like higher than utility level (long-running tasks whose progress the user does not follow actively) or background. Default seems exactly what we need to target in this case.